docs/sources/administration/configuration.md

```diff
@@ -89,9 +91,7 @@ export GF_PLUGIN_GRAFANA_IMAGE_RENDERER_RENDERING_IGNORE_HTTPS_ERRORS=true
 
 ## Variable expansion
 
-> Only available in Grafana 7.1+.
-
-> For any changes to `conf/grafana.ini` (or corresponding environment variables) to take effect, you must restart Grafana.
```

Comment:
I think that restarting grafana for the new settings to be effective is important information and it should stay unless it has been moved somewhere else in the docs.

Reply:
Yes, which is why we already mention it prominently on line 18 in this same document.